.devcontainer/Dockerfile

```diff
@@ -1,8 +0,0 @@
-ARG VARIANT="3.8-v2.0"
```

Comment:
why is this removed in this PR - is it related?

Reply:
not related. It's not in use any more since devcontainer.json update. But file wasn't removed in the past.